src/pages/dar_application/ProgressReportApplication.tsx

```diff
@@ -71,6 +123,7 @@ export const ProgressReportApplication = ({ dar, parentDar, datasets, readOnlyMo
                     }}
                     onCancel={() => {
                     }}
+                    validateForm={validateForm}
```

Comment:
I would expect the result of this function to be called on line 118 so `SubmitProgressReport` doesn't need to call back to the parent. Maybe another function that looks something like:
```
const isValidProgressReport = () => {
  return !readOnlyMode 
      && parentDar 
      && validatePRFormData(formData, datasets, dataUseTranslations);
}
```